assets/src/blocks/Covers/useCovers.js

```diff
@@ -15,14 +15,21 @@ export const useCovers = ({ post_types, tags, cover_type, initialRowsLimit, post
   const [amountOfCoversPerRow, setAmountOfCoversPerRow] = useState(null);
   const [error, setError] = useState(null);
 
+  // This method is created because Sonar Cloud does not accept
+  // repeated code. COVERS_TYPES.campaign and COVERS_TYPES.takeAction
+  // share the same settings.
+  const setCoversPerRow = () => {
+    if (layout === COVERS_LAYOUTS.carousel) {
+      setAmountOfCoversPerRow(3);
+    } else {
+      setAmountOfCoversPerRow(isSmallWindow() ? 2 : 3);
+    }
+  }
+
   const updateRowCoversAmount = () => {
     switch (cover_type) {
     case COVERS_TYPES.campaign:
-      if (layout === COVERS_LAYOUTS.carousel) {
-        setAmountOfCoversPerRow(3);
-      } else {
-        setAmountOfCoversPerRow(isSmallWindow() ? 2 : 3);
-      }
+      setCoversPerRow();
```

Comment:
Since this is now identical to the `takeAction` style, and that means there are only 2 cases left, I don't think a `switch` structure is still needed, we can probably just use an `if ... else` statement? 🤔 Or at least we can group the 2 identical cases together, such as:
```
case COVERS_TYPES.campaign:
case COVERS_TYPES.takeAction:
```

Reply:
Yep, It's a good suggestion!